app/javascript/app/document-capture/components/acuant-capture.jsx

```diff
@@ -38,24 +62,41 @@ function AcuantCapture({ label, hint, bannerText, value, onChange, className })
         </FullScreen>
       )}
       <FileInput
+        ref={inputRef}
         label={label}
         hint={hint}
         bannerText={bannerText}
         accept={['image/*']}
         value={value}
-        onClick={startCaptureIfSupported}
+        onClick={startCaptureOrTriggerUpload}
         onChange={onChange}
       />
-      {hasCapture && (
-        <Button
-          isSecondary={!value}
-          isUnstyled={!!value}
-          onClick={() => setIsCapturing(true)}
-          className="display-block margin-top-2"
-        >
-          {t(value ? 'doc_auth.buttons.take_picture_retry' : 'doc_auth.buttons.take_picture')}
-        </Button>
-      )}
+      <div className="margin-top-2">
+        {isMobile && (
+          <Button
+            isSecondary={!value}
+            isUnstyled={!!value}
+            onClick={startCaptureOrTriggerUpload}
+            className={`margin-right-${value ? 1 : 2}`}
```

Comment:
> IMO sometimes easier to see the full class names instead of interpolating?
> 
> ```
> className={value ? 'margin-right-1' : 'margin-right-2'}
> ```

Definitely something where I go back and forth depending on the circumstances ([example](https://github.com/18F/identity-idp/blob/3150ca096eaedd59f1fb1e5b5f43f93db6dd0d6d/app/javascript/app/document-capture/components/acuant-capture.jsx#L56)). I don't know that I necessarily see it as especially unreadable, though I could see the argument. Usually the bigger issue for me is whether it can be discovered via some find-in-files search. In the example linked above, I assumed it might be common to do a full search term on `doc_auth.buttons.take_picture_retry`, which wouldn't yield a result if I'd chosen to do some interpolation like ``t(`doc_auth.buttons.take_picture${ value ? '_retry' : ''}`)``. I didn't necessarily see this usage with margins as being quite as likely for that, though I suppose it's certainly possible. On the other end of the balance is if the repetition (in aggregate) might be less optimal in terms of file size, or ([like discussed in the other pull](https://github.com/18F/identity-idp/pull/4034#discussion_r467293753)) might lead to some future out-of-sync-related errors.

tl;dr: Above is some context for how I'd approached it, but in this case I think it's best to update to your suggestion. 😄

Reply:
> but in this case I think it's best to update to your suggestion.

Updated in 027bdff.